Services/KeyLoader/AbstractKeyLoader.php

```diff
@@ -51,7 +51,17 @@ public function getAdditionalPublicKeys(): array
                 throw new \RuntimeException(sprintf('Additional public key "%s" does not exist or is not readable. Did you correctly set the "lexik_jwt_authentication.additional_public_keys" configuration key?', $key));
             }
 
-            $contents[] = is_file($key) ? file_get_contents($key) : $key;
+            if (is_file($key)) {
```

Comment:
The right part of the ternary expression is missing in this refactoring. Add an `else` statement:

```php
if (is_file($key)) {
  //...
} else {
  $contents[] = $key;
}
```

Reply:
Sure, done.

Reply:
Oh crap, some tests seem broken.

Reply:
No worries, it's not related to this change :)